specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```
@@ -1359,9 +1361,249 @@
     "CacheSecuritySettings": {
       "description": "Cache security settings.",
       "properties": {
-        "rootSquash": {
+        "accessPolicies": {
+          "description": "NFS access policies defined for this cache.",
+          "items": {
+            "$ref": "#/definitions/NfsAccessPolicy"
+          },
+          "type": "array"
+        }
+      }
+    },
+    "CacheDirectorySettings": {
+      "description": "Cache Directory Services settings.",
+      "type": "object",
+      "properties": {
+        "activeDirectory": {
+          "$ref": "#/definitions/CacheActiveDirectorySettings",
+          "description": "Specifies the settings for joining the HPC Cache to an Active Directory domain."
+        },
+        "usernameDownload": {
+          "$ref": "#/definitions/CacheUsernameDownloadSettings",
+          "description": "Specifies the settings for Extended Groups. Extended Groups allows users to be members of more than 16 groups."
+        }
+      }
+    },
+    "CacheActiveDirectorySettings": {
+      "description": "Active Directory settings used to join a Cache to a Domain.",
+      "properties": {
+        "primaryDnsIpAddress": {
+          "description": "Primary DNS IP address used to resolve the Active Directory domain controller's fully qualified domain name.",
+          "type": "string"
+        },
+        "secondaryDnsIpAddress": {
+          "description": "Secondary DNS IP address used to resolve the Active Directory domain controller's fully qualified domain name.",
+          "type": "string"
+        },
+        "domainName": {
+          "description": "The fully qualified domain name of the Active Directory domain controller.",
+          "type": "string"
+        },
+        "domainNetBios": {
+          "description": "The Active Directory domain's NetBIOS name.",
+          "type": "string"
+        },
+        "smbServerName": {
+          "description": "The name (NetBIOS) used for the HPC Cache to join the Active Directory Domain. Length must not be greater than 15 and chars must be in list of [-0-9a-zA-Z_] char class.",
```

```suggestion
          "description": "The name (NetBIOS) used for the HPC Cache to join the Active Directory domain. Length must not be greater than 15 and chars must be from the [-0-9a-zA-Z] char class.",
```

Note the missing underscore from the character class as per Manuel's comment below.

---

Fixed in next commit.

---

Wording fixed. Separate comment opened about the underscore that's still there. Resolving this one.